cloudant-client/src/main/java/com/cloudant/client/org/lightcouch/CouchDatabaseBase.java

```diff
@@ -299,6 +299,53 @@ public Response remove(String id, String rev) {
     }
 
     /**
+     * Reads an attachment from the database.
+     *
+     * The stream must be closed after usage, otherwise http connection leaks will occur.
+     *
+     * @param docId the document id
+     * @param attachmentId the attachment id
+     * @return the attachment in the form of an {@code InputStream}.
+     */
+    public InputStream getAttachment(String docId, String attachmentId) {
+        final URI uri = new DatabaseURIHelper(dbUri).attachmentUri(docId, attachmentId);
```

Comment:
I think this should call down into the three arg version and default `revId` to `null` or just have the three arg version and have the caller set the `revId` to `null`.